sklearn/feature_extraction/dict_vectorizer.py

```diff
@@ -105,10 +130,18 @@ def fit(self, X, y=None):
             objects) to feature values (strings or convertible to dtype).
         y : (ignored)
 
+        Raises
```

Comment:
This is nice to document what errors are being raised for all the classes... Do you happen to know how this appears in the rendered documentation? Thanks for adding this!!